### PR TITLE
M856

### PR DESCRIPTION
**Pull request by gwan93**
https://trello.com/c/Ik7SF0q4/856-marker-states?filter=label:dash%20app

- Change appearance of clusters to be red with white border
- When a user clicks a site, update the metrics pane to show that information about that selected site
- Add a button to the top of the metrics pane selected site to clear the selected site and remove the sample_event_id from the url params
- In mobile view if a site is selected, show the site name instead of the summarized metrics
- Some of the functionality of the metrics pane when a site is selected may be missing. I just wanted to style things first based off of the mockups. Further functionality can be added in future cards

### CODE REVIEW

**Comment by gwan93** [comment]
[Marker states](https://trello.com/c/Ik7SF0q4/856-marker-states)

**Review by alanjleonard — CHANGES_REQUESTED** [review]
I think the stroke-width and the border can just be 1px.
image

Also, it crashes on me when I click a point. The screen is just white and I get these errors:
image

**Comment by gwan93** [comment]
Tagging @JBurkinshaw so reviews can be done in parallel

File: src/components/MetricsPane/MetricsPane.styles.js [hunk 1]
```diff
@@ -157,6 +165,117 @@ export const MobileExpandedMetricsPane = styled.div`
   width: 100vw;
 `
 
+export const SelectedSiteMetricsCardContainer = styled('div')`
```
Comment (JBurkinshaw) [inline src/components/MetricsPane/MetricsPane.styles.js:168]: Remember the shorthand for native html elements?
`styled.div`

**Review by JBurkinshaw — APPROVED** [review]

**Review by alanjleonard — CHANGES_REQUESTED** [review]
Update the stroke-width on the SVG single site markers to 1px. and use the attached png for the selected site.

map-pin

**Comment by gwan93** [comment]
image

**Review by alanjleonard — APPROVED** [review]

**Pull request merged**